test_esss_jenkins.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+from _pytest.pytester import LineMatcher
```

Comment:
There's no `LineMatcher` fixture (you are probably thinking about `testdir`, whose `stdout` and `stderr` attributes are `LineMatcher` instances).

I think it is fine because this is used only for testing, if it ever breaks we need to either pin the pytest version or update the test file.